### ISSUE TITLE
About evaluation results

### ISSUE THREAD

**Issue by ShoshanaXue** [prs-eth/graph-super-resolution#2]
I have downloaded the source code as well as dataset from the given links and tried to train for many times( I didn't modify any hyperparameters except for the dataset path and log path ). However, there's a large gap between my evaluation results and those in the paper.

So I would like to ask what is the problem and what should I try to get the results in the paper?

Thank you!

**Comment by riccardodelutio** [comment]
Hi, Thank you for your interest in our work! 
Could you specify for which dataset you're encountering problems?

**Comment by ShoshanaXue** [comment]
Thank you for your reply.
Actually, I tried both NYUv2 and DIML datasets. For training details, I refer to the parameters in the supplementary materials. The hyper-parameters of NYUv2 training are listed below:

key | value
-- | --
config |  
save_dir | ./out
logstep_train | 10
save_model | both
val_every_n_epochs | 1
resume |  
seed | 12345
wandb | FALSE
wandb_project | graph-sr
dataset | NYUv2
data_dir | ./dataset
num_workers | 8
batch_size | 8
crop_size | 256
scaling | 8
max_rotation | 15
no_flip | FALSE
in_memory | FALSE
loss | mse
num_epochs | 250
optimizer | adam
lr | 0.0001
momentum | 0.9
w_decay | 1.00E-05
lr_scheduler | step
lr_step | 10
lr_gamma | 0.9
skip_first | FALSE
gradient_clip | 0
feature_extractor | UResNet
pretrained | FALSE
lambda_init | 1
mu_init | 0.1

**Comment by riccardodelutio** [comment]
Ok, I see a few parameters that are not set correctly that are most likely the reason for not reproducing the results. Try changing these values:
 
| key | value |
|:------:|:-------:|
| pretrained | True |
| gradient_clip | 0.01 |
| mu_init | 0.1 |

you could also set the  in_memory to True to speed up the training.

I hope that solves your issues!

**Comment by ShoshanaXue** [comment]
Thank you for your reply, I will try it out.

**Comment by ShoshanaXue** [comment]
Thank you so much, I will try it out.

**Comment by riccardodelutio** [comment]
Just to clarify, please use the new parameters in the edited comment, I was checking the wrong run before. We will also be adding config files for all datasets to the repository soon so you will be able to directly use those to reproduce the results.
